test/integration/custom_cluster_integration_test.cc

```diff
@@ -28,7 +28,7 @@ class CustomClusterIntegrationTest : public testing::TestWithParam<Network::Addr
         [this](envoy::config::bootstrap::v3alpha::Bootstrap& bootstrap) {
           auto* cluster_0 = bootstrap.mutable_static_resources()->mutable_clusters(0);
 
-          cluster_0->clear_hosts();
+          cluster_0->clear_hidden_envoy_deprecated_hosts();
```

Comment:
Seems like this shouldn't be needed if we are no longer using the deprecated field, etc.?